apis/Google.Cloud.Diagnostics.AspNet/docs/index.md

```diff
@@ -1,26 +1,13 @@
-# Google.Cloud.Diagnostics.AspNet
+{{title}}
 
 `Google.Cloud.Diagnostics.AspNet` is an ASP.NET instrumentation library for Google Stackdriver.
```

Comment:
No - because the ASP.NET and ASP.NET Core libraries don't represent a Google product - they're just integration points.